### ISSUE TITLE
about training steps

### ISSUE THREAD

**Issue by hxm1150310617** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#178]
What's the difference between training step1 and step2?
The step1 is "training from mobilenet weights" and the step2 is "training from weights obtained from the previous step". Does is mean that I should first train from pretrained mobilenet weights for , like n epochs, and the train from this n-th model for another m epochs? If so, what's the difference between that training strategy and just train n+m epochs from pretrained mobilenet weights?

**Comment by zhenzhongle** [comment]
From the author ' s instructions, this will improve 1 point accuracy. But it takes experiments to see the effect, and I think it 's a break training. For example, my computer power failure or failure leads to suspension, and I can come back from the new.

**Comment by Daniil-Osokin** [comment]
Hi! This is just a training restart. It was left for legacy reasons and reproducibility. I believe other more straightforward training strategies exist.

**Comment by hxm1150310617** [comment]
OK I get it! Thank you both for your replies.

**Comment by Daniil-Osokin** [comment]
Ok, closing.
